backend/app/agents/devrel/github/services/github_mcp_client.py

```
@@ -0,0 +1,105 @@
+                if response.status == 200:
+                    result = await response.json()
+                    if result.get("status") == "success":
+                        return result.get("data", {})
+                    else:
+                        return {"error": result.get("error", "Unknown error")}
+                else:
+                    logger.error(f"MCP server error: {response.status}")
+                    return {"error": f"MCP server error: {response.status}"}
+
+        except aiohttp.ClientError as e:
+            logger.error(f"Error communicating with MCP server: {e}")
```

Ruff (0.12.2)

Use `logging.exception` instead of `logging.error`

Replace with `exception`

(TRY400)